pimd/pim_instance.c

```diff
@@ -148,7 +148,7 @@ static int pim_vrf_new(struct vrf *vrf)
 {
 	struct pim_instance *pim = pim_instance_init(vrf);
 
-	zlog_debug("VRF Created: %s(%u)", vrf->name, vrf->vrf_id);
+	zlog_info("VRF Created: %s(%u)", vrf->name, vrf->vrf_id);
```

Comment:
What do we expect the end users to do with this information?  Do they care?  I would say this just needs to be removed or debug guarded